biorxiv/corpora_comparison/figure_generator.py

```diff
@@ -0,0 +1,409 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Figures for Corpora Comparison between bioRxiv,  Pubmed Central, New York Times
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import numpy as np
+import pandas as pd
+
+from cairosvg import svg2png
+from IPython.display import Image
+import plotnine as p9
+
+from annorxiver_modules.corpora_comparison_helper import(
+    calculate_confidence_intervals,
+    create_lemma_count_df,
+    plot_pointplot,
+    plot_bargraph,
+    plot_point_bar_figure
+)
+
+
+# In[2]:
+
+
+subset = 20
+
+
+# # KL Divergence Graph
+
+# In[3]:
+
+
+kl_divergence_df = pd.read_csv(
+    "output/comparison_stats/corpora_kl_divergence.tsv", 
+    sep="\t"
+)
+kl_divergence_df.head()
+
+
+# In[4]:
+
+
+g = (
+    p9.ggplot(
+        kl_divergence_df.replace({
+        "biorxiv_vs_nytac":"biorxiv_vs_reference",
+        "pmc_nytac": "pmc_vs_reference"
```

Comment:
`pmc_vs_nytac` is still showing up on this figure which is confusing

```suggestion
        "pmc_vs_nytac": "pmc_vs_reference"
```